src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TeamsController.cs

```diff
@@ -351,5 +334,67 @@ private string PhotoPath(int id)
             var photoPath = string.Format(Constants.TEAM_PHOTO_PATH, id);
             return _fileService.FileExists(HttpContext.Request.MapPath(photoPath)) ? photoPath : string.Format(Constants.TEAM_PHOTO_PATH, 0);
         }
+
+
+        private void UpdateRoaster(TeamViewModel teamViewModel, Team team, IEnumerable<int> playersInTeamDb)
+        {
+            //select players which are in Db
+            var registeredPlayers = teamViewModel.Roster
+                .Where(x => x.Id != 0)
+                .ToList();
+            //select teams which players returned from view
+            var playersTeams = registeredPlayers
+                .Select(x => _playerService.GetPlayerTeam(_playerService.Get(x.Id)))
+                .ToList();
+            //check if players play in another team
```

Comment:
This is business logic what is it doing here?
 Also i think you don't need this whole method
Ask Sofiia - she knows how to do this. We talked about it in details

Reply:
Because there is no other method in Service which has the same functionality. Controller method calls 'AddPlayers', 'RemovePlayers' an 'ChangeCaptain.' 'AddPlayers' doesn't remove, 'RemovePlayers' doesn't add players. 

Which is why we need to find out first what are we going to pass to these methods. So only way now is to use this method in Controller itself.

Reply:
Don't you know on the UI level which teams were added and which were removed by user?